openmdao/core/problem.py

```diff
@@ -869,15 +863,16 @@ def setup(self, check=False, logger=None, mode='auto', force_alloc_complex=False
             'connections': {},
             'remote_systems': {},
             'remote_vars': {},  # does not include distrib vars
-            'prom2abs': {'input': {}, 'output': {}}  # includes ALL promotes including buried ones
+            'prom2abs': {'input': {}, 'output': {}},  # includes ALL promotes including buried ones
+            '_setup_status': _SetupStatus.PRE_SETUP
```

Comment:
can you get rid of the underscore on `_setup_status` to make it consistent with the other members of `_metadata`?

Reply:
Fixed